src/database.py

```diff
@@ -6,5 +6,5 @@
 
 file_name = "ca-certificate.crt" if os.getenv("STAGE") == "local" else "/etc/ssl/ca-certificate.crt"
 
-client = MongoClient(os.getenv("MONGO_URI"), tls=True, tlsCAFile=file_name)
-db = client[os.getenv("MONGO_DB")]
```

Comment:
This is necessary for production. Do this:

```
if os.getenv("STAGE") == "local":
    file_name = "ca-certificate.crt"
    use_tls = os.getenv("MONGO_URI") != "mongodb://localhost:27017/"
else:
    file_name = "/etc/ssl/ca-certificate.crt"
    use_tls = True
client = MongoClient(os.getenv("MONGO_URI"), tls=use_tls, tlsCAFile=file_name)
db = client[os.getenv("MONGO_DB")]
```
Essentially if you are connecting to your own mongo database when it will be local and not use tls but if you or the server is connecting to the appdev hosted mongo database then we need to use tls and have that file in your directory